src/main/java/vmware/samples/vcenter/vm/create/basicvm/CreateBasicVM.java

```diff
@@ -63,28 +64,35 @@
     protected void parseArgs(String[] args) {
         Option datacenterOption = Option.builder()
             .longOpt("datacenter")
-            .desc("The name of the datacenter on which to create the vm.")
+            .desc("The name of the datacenter in which to create the vm.")
```

Comment:
Datacenteres would hold the Clusters, servers etc. A VM would would be hosted in it.